### DISCUSSION TITLE
How to forget a lot of series at once

### DISCUSSION THREAD

**Discussion by effemmeffe** [Flexget/Flexget#4072] [Q&A]
I have a lot of series in my database and I'd like to remove almost all of them.
With "flexget series list" I've got a file with all the series I don't want anymore. Is there a CLI command that I can use?
My config reads my wanted series from a trakt list, I wonder if it would be better to wipe out the whole database and then run the config from scratch instead of trying to forget a whole bunch of series (in this case which CLI command should I run?)
Please advice, thanks.

**Comment by gazpachoking** [comment]
Probably easiest is to reset the database for the series plugin, `flexget database reset-plugin series`, then set the begin episodes for the series you didn't want to reset `flexget series begin "some show (2019)" S06E01`

**Reply by effemmeffe** [reply]
Ok, I'll give it a try, thanks.
